multiversx_sdk_cli/cli_deps.py

```diff
@@ -34,16 +36,63 @@ def install(args: Any):
 
 def check(args: Any):
     name: str = args.name
-    module = dependencies.get_module_by_key(name)
-    tag_to_check: str = config.get_dependency_tag(module.key)
+
+    if name == "all":
+        all_dependencies = dependencies.get_all_deps()
+        missing_dependencies: List[Tuple[str, str]] = []
+
+        for dependency in all_dependencies:
+            tag_to_check: str = config.get_dependency_tag(dependency.key)
+            is_installed = check_module_is_installed(dependency, tag_to_check)
+
+            if not is_installed:
+                missing_dependencies.append((dependency.key, tag_to_check))
+
+        if len(missing_dependencies):
+            raise errors.DependenciesMissing(missing_dependencies)
+        return
+    if name == "rust":
```

Comment:
Nice for having the check done in this new, better way.

Reply:
This is now refactored.